packages/helm-charts/blockscout/scripts/grafana-hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+echo Creating Grafana tag
+export CELO_ENV=$CELO_ENV
+export SUFFIX=$SUFFIX
+
+REQUEST=$(curl "$GRAFANA_API_ENDPOINT/api/annotations" \
+  --request "POST" \
+  --header "authorization: Bearer $GRAFANA_API_TOKEN" \
+  --header "content-type: application/json" \
+  --data @- << EOF
+  {
+    "text": "Deployed $CELO_ENV $SUFFIX by $GCLOUD_ACCOUNT with commit: \n \n <a href=\"https://github.com/celo-org/blockscout/commit/$TAG\"> $TAG</a>\n ",
```

Comment:
As agreed, let's pass this string from the outside of the script, you can use helm `.Release.Name` instead of `$CELO_ENV $SUFFIX` as those are unavailable.